script.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-unused-vars */
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+const addBtn = document.getElementById('add-btn');
+const booksContainer = document.querySelector('.books-container');
+const getLocalBooks = localStorage.getItem('books');
+const localBooks = JSON.parse(getLocalBooks);
+let booksCollection = getLocalBooks ? localBooks : [];
+
+function addBook(title, author) {
+  const book = {
+    title,
+    author,
+  };
+  booksCollection.push(book);
+  localStorage.setItem('books', JSON.stringify(booksCollection));
+}
+
+function handleChange() {
+  addBook(bookTitle.value, bookAuthor.value);
+}
+
+function removeBook() {
+  const bookClass = window.event.target.className.split('-')[1];
+  document.querySelector(`.div-${bookClass}`).style.display = 'none';
+  const bookId = window.event.target.id;
+  const newbooksCollection = booksCollection.filter((book) => book.title !== bookId);
```

Comment:
- [ ] I will suggest using another property to filter the books, something like an auto-generated `id`, using the `title` will create issues if you have two books with the same `title` but different `author`, kindly change this 😄